### ISSUE TITLE
Pair directly with the lights and intercept commands from the remote

### ISSUE THREAD

**Issue by torrottum** [torrottum/ikea-ansluta-esphome#2]
Turning on the lights using Home Assistant is actually a bit more reliable than using the IKEA remote (sometimes not all lights turn on/off). I think this is because this send the commands 200 times while the remote sends them 50 times.

Another issue is that the remote keeps its own state.  If the lights are off and I click the remote 3 times the lights will go 50%, 100%, 50%. If I then turn off the lights with HA I need to press the remote 2 times to turn the lights on again. Because the remote sends off and then 50%.

**Comment by torrottum** [comment]
I think I have a good solution for pairing now. Since we have proper configuration after #7 is done it's possible to do this:
```yaml
light:
  - platform: ikea_ansluta
    name: 'IKEA Ansluta'
    address: 0xfefe # Optional address to pair against the lights with
    remote_address: 0x35c0
    remote_mode: 'default' # not sure if this should be a string or a list of the sequence you want
```
To pair with the light:
1. Intercept the pair command from the remote 
2. wait for example 5 seconds
3. send the pairing command with the specified address. 
That waiting should give you enough time to enable pairing mode on the Utrusta/Omlopp (I have no idea what light I actually have) light.

When sending commands to the light, we can use address if that's set or else we fallback to the remote address. 

For the issue of the remote keeping its own state we just ignore the command from the remote and use our state.
The `remote_mode` configuration can be used to set how we want the remote to behave. For example `default` for the default sequence: 50%, 100%, 50%, Off, or `on_off` for toggling the state.
